internal/broker/broker.go

```diff
@@ -435,7 +457,7 @@ func (b *Broker) handleIsAuthenticated(ctx context.Context, session *sessionInfo
 		return AuthRetry, errorMessage{Message: "could not decode challenge"}
 	}
 
-	var authInfo authCachedInfo
+	var authInfo token.AuthCachedInfo
```

Comment:
Also, how do you feel about splitting this? It used to be the same type to make things easier+slightly faster when decrypting all stuff. Now that we don't have encryption, maybe we can split the token from the cached user information? i.e. having a user`.token` and a `user.info` file, for example... It would require a bit more refactoring in the code, so I'm down to just open a card for it and then we can tackle it in another PR.

Reply:
I also considered that but didn't do it because I think it might still simplify things to have the secrets in a single file (well, it's already two now with the password in a separate file) when we use use systemd credentials to protect them (see https://github.com/ubuntu/authd/issues/472).

I would prefer to go with what I proposed in https://github.com/ubuntu/authd-oidc-brokers/pull/134#discussion_r1757031321, i.e. use the term "token" for the whole JSON document we store persistently, which includes the OAuth2 token and the user info.

Reply:
Sounds good to me!